src/features/sidebar/components/Sidebar.jsx

```diff
@@ -6,10 +6,10 @@ import SidebarItem from "@features/sidebar/components/SidebarItem";
 //demo data
 import categories from "@demo-data/demoCategories.json";
 
-const Sidebar = () => {
+const Sidebar = ({ classes }) => {
   return (
-    <aside className="h-screen w-80">
-      <nav className="h-full flex flex-col bg-white-500 border-e shadow-sm w1/2">
+    <aside className={`${classes ? classes : 'hidden md:block'}`}>
```

Comment:
if u want you can simplify the syntax ${classes ? classes : 'hidden md:block'} by using the logical OR operator. The shorthand version would be {classes || 'hidden md:block'}. This achieves the same result more efficiently.